src/main/java/com/azure/cosmos/kafka/connect/CosmosDBConfig.java

```diff
@@ -32,12 +33,16 @@
         + "For example: topic1#con1,topic2#con2.";
     private static final String COSMOS_CONTAINER_TOPIC_MAP_DISPLAY = "Topic-Container map";
 
+    public static final String  COSMOS_PROVIDER_NAME_CONF = "connect.cosmos.provider.name";
```

Comment:
how would differentiating between different apps work? do we need to differentiate the apps?

Reply:
discussed offline. in future if we see the need, we can add another config later for app name (outside of the scope of this PR).